src/arch/utils/mainparam_utils.cpp

```diff
@@ -9,17 +10,24 @@
  * Contributors:
  *   Tarik Terzimehic
  *    - initial API and implementation and/or initial documentation
+ *   Ernst Blecha
+ *    - add command line option to enable fakeTimer
  *******************************************************************************/
 
 #include <forte_config.h>
 #include <stdio.h>
 #include <string.h>
 #include <stdlib.h>
+#include "../core/forteinstance.h"
```

Comment:
do you actually need this?

Reply:
Thank you - this should have been devlog.h